### ISSUE TITLE
Unexpected behavior when importing  a schema

### ISSUE THREAD

**Issue by nikitaulshin** [ardatan/graphql-import#133]
Hello!
I'm using your awesome library to split my server-sice (NodeJS & TypeScript) code.
Unfortunately I faced an issue with import.

I have this .graphql file:

```
type Query {
    users: String
}

type AuthMutation {
    authenticate(code: String!, accountId: String): String
}

schema {
    query: Query,
    mutation: AuthMutation
}
```

And I'm trying to import it in this way: 

```js
import { importSchema } from 'graphql-import';
import * as path from 'path';

const AuthSchema = importSchema(path.join(__dirname, './auth.graphql'));
```

But I'm getting an incorrect result: 

```
type Query {
    users: String
}

type AuthMutation {
    authenticate(code: String!, accountId: String): String
}
```

No schema. Any suggestions on this?

Thank you!

**Comment by ardatan** [comment]
Hi @nikitaulshin !
In 1.0.0 beta release, we introduced a lot of changes including schema definition support.
Could you install graphql-import@beta to try new changes? Don't forget to modify your code regarding to the migration notes in README.
https://github.com/ardatan/graphql-import#updating-from-07x

Let us know if your issue still persists.

**Comment by ardatan** [comment]
Available in 1.0.0!
